.travis.yml

```diff
@@ -48,14 +48,13 @@ install:
   # pyenv's shims directory is not in the PATH by default.
   - export PATH="$(pyenv root)/shims:${PATH}"
   # These are the Python versions that we want to test on.
-  - pyenv install --skip-existing ${PY34}
   - pyenv install --skip-existing ${PY35}
   - pyenv install --skip-existing ${PY36}
   - pyenv install --skip-existing ${PY37}
-  - pyenv global ${PY37} ${PY36} ${PY35} ${PY34} system
+  - pyenv global ${PY37} ${PY36} ${PY35} system
   # tox is not installed by default in some older Travis environments.
   - python3.6 -m pip install tox
```

Comment:
I think the manual installation of `tox` is no longer necessary now that we don't use the `xcode6.4` image anymore. On `xcode7.3` (now the oldest image we use), you can see that [`tox` is already installed and the `pip` call does nothing](https://travis-ci.org/pybee/rubicon-objc/jobs/506412490#L2202).

Reply:
I did __pip install --upgrade tox__ to make sure we are running the latest and greatest.